config/initializers/session_store.rb

```diff
@@ -1,8 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-Elovation::Application.config.session_store :cookie_store, key: '_elovation_session'
-
```

Comment:
I'm not sure, honestly. The rails upgrade script changed all these and the app still ran so I kept it.